lib/game_of_life_web/templates/universe/show.html.leex

```diff
@@ -20,33 +22,24 @@
 
         <%= label class: "form-label inline middle" do %>20<% end %>
       </div>
+      <% end %>
+    </div>
 
-      <p class="help-text">Current speed: <%= @speed %></p>
-    <% end %>
-  </div>
 
   <%= form_tag "#", phx_change: "set_template" do %>
-    <div class="form-control">
-      <%= label class: "form-label inline middle" do %>Select Template: <% end %>
-
-      <%= select :universe, :template, template_names() %>
+    <div class="form-group">
+      <%= label :universe, :template %>
+      <%= select :universe, :template, template_names(), selected: @template %>
     </div>
-
-    <p class="help-text">Current template: <%= @template %></p>
   <% end %>
 
   <div class="form-group">
-    <%= form_tag "#", class: "inline", phx_submit: "toggle_playing" do %>
-      <%= submit (if @playing, do: "Pause", else: "Play"), class: "button" %>
-    <% end %>
-
-    <%= form_tag "#", class: "inline", phx_submit: "reset" do %>
-      <%= submit "Reset", class: "button" %>
-    <% end %>
+    <%= link play_text(@playing), to: "#", phx_click: "toggle_playing", class: "button" %>
```

Comment:
Would we be able to use an actual `<button>` here directly, rather than a link?

I forgot that `phx_click` was a thing, hence abusing `phx_submit`

Reply:
if we use the elixir button helper it actually generates a form